smtp:
  server: 'smtp.gmail.com'
  port: 25
  use_authentication: true
  username: 'sender@domain.com'
  password: '12345678'
  sender: 'sender@domain.com'
  default_recipients:
    - 'recipient@domain.com'
    - 'recipient@domain.com'